e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -77,9 +81,11 @@ test.describe('Example Imagery', () => {
 
     test('Can use alt+drag to move around image once zoomed in', async ({ page }) => {
         const deltaYStep = 100; //equivalent to 1x zoom
+        const panHotkey = process.platform === 'linux' ? ['Control', 'Alt'] : ['Alt'];
```

Comment:
@johnhill2 - Disappointingly Playwright failed to parse a modifier sequence like `Control+Alt` so the test had to be modified to map over an array of key presses.